### PR TITLE
Core/continue removing reasonml part 3

### PR DESCRIPTION
**Pull request by rej156**

### CODE REVIEW

File: packages/kauri-components/components/Outline/index.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+const UserContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  > :first-child {
+    margin-bottom: ${props => props.theme.space[1]}px;
+  }
+`;
+
+const RuntimeProps = t.interface({
```
Comment (davodesign84) [inline packages/kauri-components/components/Outline/index.tsx]: What's this malarky?

File: packages/kauri-components/lib/theme-config.ts [hunk 2]
```diff
@@ -1,4 +1,8 @@
-import theme, { IThemeInterface } from "../../kauri-web/lib/theme-config";
+import theme, {
+  IThemeInterface,
+  metamask,
+  categories,
```
Comment (davodesign84) [inline packages/kauri-components/lib/theme-config.ts]: do we still need categories in theme-config?

File: packages/kauri-components/package.json [hunk 3]
```diff
@@ -2,12 +2,9 @@
   "name": "kauri-components",
   "version": "1.0.0",
   "scripts": {
-    "build": "bsb -make-world",
-    "dev:bs": "bsb -make-world -w",
-    "clean": "bsb -clean-world",
     "start:storybook": "STORYBOOK=true start-storybook -p 6006",
-    "storybook": "concurrently \"npm run build\" \"STORYBOOK=true npm run start:storybook\"",
-    "dev": "concurrently \"npm run dev:bs\" \"STORYBOOK=true npm run start:storybook\"",
+    "storybook": "STORYBOOK=true npm run start:storybook",
```
Comment (davodesign84) [inline packages/kauri-components/package.json]: 🏆

File: packages/kauri-components/package.json [hunk 4]
```diff
@@ -22,14 +19,10 @@
   "license": "MIT",
   "esy": {},
   "dependencies": {
-    "@glennsl/bs-revamp": "^0.1.1",
     "@storybook/addon-knobs": "4.0.7",
     "@storybook/addons": "4.0.7",
     "@storybook/core": "4.0.7",
     "@storybook/react": "4.0.7",
-    "bs-css": "^7.0.1",
```
Comment (davodesign84) [inline packages/kauri-components/package.json]: 🤤

File: packages/kauri-web/components/connections/Community/Community.tsx [hunk 5]
```diff
@@ -112,7 +113,7 @@ const Container: React.SFC<Props> = props => {
   });
   return (
     <section>
-      {/* <ScrollToTopOnMount></ScrollToTopOnMount> */}
+      <ScrollToTopOnMount />
```
Comment (davodesign84) [inline packages/kauri-web/components/connections/Community/Community.tsx]: Should we not just remove this, Nelson keeps complaining about this

File: packages/kauri-web/components/containers/Collection/View.js [hunk 6]
```diff
@@ -43,61 +43,98 @@ const HeaderContainer = styled(ContentContainer)`
...
+    const bg = (background && background) || this.state.trianglifyBg;
+    const url = `https://${hostName.replace(/api\./g, '')}/collection/${
+      this.props.id
+    }/${slugify(name, {
       lower: true,
-    })}`
+    })}`;
 
-    console.log(JSON.stringify(this.props.data.getCollection))
+    // console.log(JSON.stringify(this.props.data.getCollection));
```
Comment (davodesign84) [inline packages/kauri-web/components/containers/Collection/View.js]: remove?

File: packages/kauri-web/components/containers/Discover/Communities/View.js [hunk 7]
```diff
@@ -0,0 +1,138 @@
+// @flow
```
Comment (davodesign84) [inline packages/kauri-web/components/containers/Discover/Communities/View.js]: Why reintroducing this file?

File: packages/kauri-components/components/Outline/index.tsx (hunk 1 above)
Comment (rej156) [reply packages/kauri-components/components/Outline/index.tsx]: https://lorefnon.tech/2018/03/25/typescript-and-validations-at-runtime-boundaries/

File: packages/kauri-components/lib/theme-config.ts (hunk 2 above)
Comment (rej156) [reply packages/kauri-components/lib/theme-config.ts]: For stories mock data apparently

File: packages/kauri-web/components/connections/Community/Community.tsx (hunk 5 above)
Comment (rej156) [reply packages/kauri-web/components/connections/Community/Community.tsx]: But he was the one that wanted it in
Wtf man

File: packages/kauri-web/components/containers/Discover/Communities/View.js (hunk 7 above)
Comment (rej156) [reply packages/kauri-web/components/containers/Discover/Communities/View.js]: git pull funneh

File: packages/kauri-web/components/containers/Collection/View.js (hunk 6 above)
Comment (rej156) [reply packages/kauri-web/components/containers/Collection/View.js]: Dem debug lines 
Will remove

File: packages/kauri-web/components/connections/Community/Community.tsx (hunk 5 above)
Comment (rej156) [reply packages/kauri-web/components/connections/Community/Community.tsx]: Will remove then, apparently we want it in the article though methinks

File: packages/kauri-components/components/Outline/index.tsx (hunk 1 above)
Comment (rej156) [reply packages/kauri-components/components/Outline/index.tsx]: tl;dr
io-ts introduces the concept of a runtime type (an instance of the Type class) which represents a runtime validator for a typescript (static) type.

Comment (rej156) [reply packages/kauri-components/components/Outline/index.tsx]: https://www.olioapps.com/blog/checking-types-real-world-typescript/
Better read*

**Pull request merged**